recipes/mp_pytorch/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "mp_pytorch" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mp_pytorch-{{ version }}.tar.gz
+  sha256: d20aba17454016f9b5548c90356bb56a1f05a5a2ac0944d08c3da0ed707e6dbe
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - matplotlib-base
+    - pytorch
+    - addict
+
+test:
+  imports:
+    - mp_pytorch
```

Can you add a few more imports here? Their `__init__` is only defining `__version__` so it's not covering much.

For example:

```suggestion
    - mp_pytorch
    - mp_pytorch.mp
    - mp_pytorch.demo.demo_basis_gn
```